tutorials/using-cloud-vpn-with-alibaba-with-redundancy

```diff
@@ -0,0 +1,950 @@
+instance in a VPC, you must specify the VSwitch that the instance is located.
+-  **Alibaba Cloud Zone**-Zones are physical areas with independent power grids 
+and networks in one region. Alibaba recommends creating different VSwitches in 
+different zones to achieve disaster Recovery.
+-  **Alibaba Cloud VPN Gateway**-The VPN gateway is the IPsec VPN gateway created 
+on the Alibaba Cloud side. One VPN gateway can have multiple VPN connections.
+-  **Alibaba Cloud Customer Gateway**-The customer gateway is the VPN service 
+deployed in the on-premises data Center or, in this case, the GCP Cloud VPN gateway. 
+By creating a customer gateway, you can register the VPN information to the cloud, 
+and then create a VPN connection to connect the VPN gateway and the customer gateway.
+-  **Alibaba Cloud VRouter**-A VRouter is a hub in the VPC that connects all 
+VSwitches in the VPC and serves as a gateway device that connects the VPC to 
+other networks. VRouter routes the network traffic according to the configurations 
+of route entries.
+-  **Alibaba Cloud Route Entry**-A route entry specifies the next hop address for the 
+network traffic destined to a CIDR block. It has two types of entries: system route 
+entry and custom route entry.
+-  **Alibaba Cloud Route Table**-A route table is a list of route entries in a VRouter.
+
+## Topology
+
+Cloud VPN supports the following topology with Alibaba Cloud VPN Gateway:
+
+-  Redundant IPsec VPN tunnel configuration using static routing.
+
+<p><img align="center" src="topology-redundancy.PNG" height="200" /></p>
+
+For detailed topology information, see the following resources:
+
+-  For basic VPN topologies, see 
+[Cloud VPN Overview](https://cloud.google.com/vpn/docs/concepts/overview).
+-  For redundant topologies,  the
+[Cloud VPN documentation on redundant and high-throughput VPNs](https://cloud.google.com/vpn/docs/concepts/redundant-vpns).
+
+Disclaimer: At time of this writing, IPsec VPN tunnel configuration using dynamic routing between Cloud VPN and Alibaba Cloud VPN Gateway is not supported. 
+
+## Product environment
+
+The on-premise VPN gateway used in this guide is as follows:
+
+-  Vendor—Alibaba Cloud
+-  Service—VPN Gateway
+
+## Before you begin
+
+Follow the steps in this section to prepare for VPN configuration.
+
+**Note**: This guide assumes that you have basic knowledge of the
+[IPsec](https://wikipedia.org/wiki/IPsec) protocol.
+
+### GCP account and project
+
+Make sure you have a GCP account. When you begin, you must select or create a
+GCP project where you will build the VPN. For details, see
+[Creating and Managing Projects](https://cloud.google.com/resource-manager/docs/creating-managing-projects).
+
+### Permissions
+
+To create a GCP network, a subnetwork, and other entities described in this
+guide, you must be able to sign in to GCP as a user who has
+[Network Admin](https://cloud.google.com/compute/docs/access/iam#network_admin_role)
+permissions. For details, see
```

Comment:
change: For details, see
\[Required Permissions]\(https://cloud.google.com/vpn/docs/how-to/creating-vpn-dynamic-routes#required_permissions)
in the article
\[Creating a VPN Tunnel using Dynamic Routing]\(https://cloud.google.com/vpn/docs/how-to/creating-vpn-dynamic-routes).

to: For details, see the \[Required Permissions]\(https://cloud.google.com/vpn/docs/how-to/creating-vpn-dynamic-routes#required_permissions) section in the \[Creating a VPN Tunnel using Dynamic Routing]\(https://cloud.google.com/vpn/docs/how-to/creating-vpn-dynamic-routes) article.